### ISSUE TITLE
Section called "Accessibility standards conformance"

### ISSUE THREAD

**Issue by LaZay** [mattgarrish/epub-revision#2407]
The technical spec does not tell how to specify the accessibility standards conformance (except for [ARIA](https://www.w3.org/TR/epub-a11y-tech-11/#meta-006)).
https://www.w3.org/TR/epub-a11y-tech-11/#sec-discovery

Only the functional spec does. It explains how to indicate the A11Y accessible epub spec version + WCAG version + WCAG level, thanks to a string pattern: [A11Y-VER](https://www.w3.org/TR/epub-a11y-11/#acc-ver) + [WCAG-VER](https://www.w3.org/TR/epub-a11y-11/#wcag-ver) + [WCAG-LVL](https://www.w3.org/TR/epub-a11y-11/#wcag-lvl)
With OPF examples. 
https://www.w3.org/TR/epub-a11y-11/#sec-conf-reporting-pub

Maybe the OPF examples should move from the functional spec to the technical one? And the [3.7 Identify ARIA conformance](https://www.w3.org/TR/epub-a11y-tech-11/#meta-006) section should be enlarged to a _3.7 Accessibility standards conformance_?

**Comment by mattgarrish** [comment]
ARIA doesn't define conformance so I'd be leery of anyone claiming that they do conform.

accessibilityAPI is now removed exactly because it strains the idea of conforming to ARIA (or more accurately, being compatible with it). Unless you put an attribute on a wrong element or use an invalid value, your content conforms to the ARIA specification. It doesn't mean your content is necessarily any more accessible, though, as that's a separate issue.

It really doesn't mean much for content to be "compatible" with ARIA, in other words. It's the reading system that exposes the markup through the accessibility tree. You can claim to have used ARIA in your content, which is what accessibilityFeature now allows, but that's all. WCAG is a better measure of whether you've achieved some beneficial result from the markup since it defines outcomes that need to be attained.

As to moving the examples, I'm not sure I see a lot of benefit. I believe we discussed it at some point (possibly in the ISO revision), but there's nothing more to say about the conformance metadata than is already expressed in the requirements. It's pretty straightforward stuff. Splitting off the examples only makes reading the specification more complex as you have to flip documents, or leaves a lot of duplication to maintain.

Unlike the discovery metadata, which isn't restricted to schema.org (although practically it is), the conformance metadata is also specific to the package document.

**Comment by LaZay** [comment]
OK for ARIA. That's non sense indeed!

**Comment by iherman** [comment]
Comment with my admin hat on: @LaZay, does the answer satisfy you? If so, can this issue be closed? Thanks.

**Comment by LaZay** [comment]
Most developpers do not read the functional spec, and only refer to the technical spec. So they might miss this VERY important information.
What about moving the examples from the functional spec to the technical spec?  In the technical spec, there is only this [3.7 Identify ARIA conformance](https://www.w3.org/TR/epub-a11y-tech-11/#meta-006) section so far (for the API). There is nothing to _Identifiy EPUB A11Y spec conformance,_ nor to _Identify WCAG conformance_. Which is ironic, is'nt it?

**Comment by mattgarrish** [comment]
Please keep in mind that the techniques document is a note, not a specification. It provides non-normative guidance on meeting some of the requirements of the specification, where those requirements are unique to publications.

If you're reading the techniques document with the idea that it's all that you have to do to be accessible, then you're starting from an incorrect assumption.

**Comment by LaZay** [comment]
Thanks for that reminder.

Just to let you know, the French community wanted to translate EPUB A11Y specs in French, in order to help people work on accessibility. The SNE hesitated between the two docs (["functional"](https://idpf.org/epub/a11y/accessibility-20170105.html)/[technical](https://idpf.org/epub/a11y/techniques/techniques.html)), but finally decided for the technical one, for three reasons:
- The subcontrators (EPUB creators) would do the job;
- Editors-publishers would never read any of the 2 specs;
- 2 translations into French was too much work for the [SNE](https://www.sne.fr/numerique-2/normes-et-standards/).

The SNE asked IDPF's permission.  Here is the resulting French version hosted by EdrLab: [Techniques d’Accessibilité EPUB 1.0](http://www.edrlab.org/public/sne/TAE_HTML_V3/Techniques_d_Accessibilite_EPUB%201.0.htm), in which I actively participated.

Because it was translated, I am afraid most of us forgot about the different status of this tech doc:
-  _Informational Document_, as opposed to the _Recommended Specification_ in [1.0](https://idpf.org/epub/a11y/techniques/techniques.html);
-  _Group note_, as opposed to _Candidate Recommendation Draft_ so far, soon a _Recommended Specification_ in [1.1](https://www.w3.org/TR/epub-a11y-tech-11/).

Usually technical docs come with functional ones (as for the WCAG typically). The configuration a slightly different here.

In the [EPUB Accessibility Techniques 1.1](https://www.w3.org/TR/epub-a11y-tech-11), in the section [Status of This Document](https://www.w3.org/TR/epub-a11y-tech-11/#sotd) then, could you just add an explicit reference to the [True spec](https://www.w3.org/TR/epub-a11y-11/), and remind what is the relationship between the two documents? Also insist that the tech note is NOT self-sufficient, and MUST be completed with the [Conformance and Discoverability Requirements](https://www.w3.org/TR/epub-a11y-11/)? That would be great!
